automation/test-execution/scripts/bash/run-mteb-missing-tests.sh

```
@@ -0,0 +1,142 @@
+    "RedHatAI/Qwen3-Embedding-8B|reranking|80-90 min|false"
+    "RedHatAI/Qwen3-Embedding-8B|pair_classification|45-60 min|false"
+)
+
+total_tests=${#TESTS[@]}
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "Tests to run: $total_tests"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+for test in "${TESTS[@]}"; do
+    IFS='|' read -r model preset time trust_code <<< "$test"
+    if [ "$trust_code" = "true" ]; then
+        echo "  - $(basename $model): $preset (~$time) [trust-remote-code]"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)